cla_public/apps/callmeback/views.py

```diff
@@ -46,6 +46,7 @@ def request_callback():
 @callmeback.route('/result/confirmation')
 @redirect_if_no_session()
 def confirmation():
-    response = render_template('result/confirmation.html')
+    response = render_template('result/confirmation.html',
+        eligibility_status=session.eligibility_status)
```

Comment:
The reason it's passed explicitly is because the session is cleared on the following line.